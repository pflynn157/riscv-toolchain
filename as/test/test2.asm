add x10, x20, x30
sub x10, x20, x30
sll x10, x20, x30
slt x10, x20, x30
sltu x10, x20, x30
xor x10, x20, x30
srl x10, x20, x30
sra x10, x20, x30
or x10, x20, x30
and x10, x20, x30
