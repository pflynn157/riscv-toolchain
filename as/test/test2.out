01ea0533
41ea0533
01ea1533
01ea2533
01ea3533
01ea4533
01ea5533
41ea5533
01ea6533
01ea7533
